src/wallet/wallet2_api.h

```diff
@@ -662,11 +662,20 @@ struct WalletManager
     /*!
      * @brief TODO: delme walletExists - check if the given filename is the wallet
      * @param path - filename
-     * @return
+     * @return - true if wallet exists
      */
     virtual bool walletExists(const std::string &path) = 0;
 
     /*!
+     * @brief verifyWalletPassword - check if the given filename is the wallet
+     * @param keys_file_name - location of keys file
+     * @param password - password to verify
+     * @param watch_only - verify only view keys?
+     * @return - true if password is correct
+     */
+    virtual bool verifyWalletPassword(const std::string &keys_file_name, const std::string &password, bool watch_only) = 0;
```

Comment:
I mentioned before this could be const. Is it not actually possible in practice ?